executor/src/witgen/rows.rs

```diff
@@ -458,6 +487,7 @@ impl<'row, 'a, T: FieldElement> RowPair<'row, 'a, T> {
     pub fn get_value(&self, poly: &AlgebraicReference) -> Option<T> {
         let row = self.get_row(poly.next);
         if self.unknown_strategy == UnknownStrategy::Zero {
+            // TODO this should panic if the column is out of range.
```

Comment:
Ah no, I still wanted to look into which calls to get_value should panic and which should return  None.